cli/lsp/tsc.rs

```diff
@@ -4026,11 +4027,11 @@ fn op_load<'s>(
 }
 
 #[op2]
+#[serde]
 fn op_resolve<'s>(
-  scope: &'s mut v8::HandleScope,
   state: &mut OpState,
   #[serde] args: ResolveArgs,
-) -> Result<v8::Local<'s, v8::Value>, AnyError> {
+) -> Result<Vec<Option<(String, String)>>, AnyError> {
```

Comment:
We made this return a v8 local to include the serialization in the performance mark, I reverted that to make it more easily unit testable with `op_resolve::call()`. Not sure it was particularly helpful anyway.

Reply:
That's fine 👍